include/triton/Dialect/Triton/IR/TritonAttrDefs.td

```diff
@@ -134,4 +135,45 @@ def TT_ScaleDotElemTypeAttr : I32EnumAttr<
   let cppNamespace = "::mlir::triton";
 }
 
+def TT_MakeDescriptorAttr : AttrDef<Triton_Dialect, "MakeDescriptor"> {
+  let attrName = "triton.gpu.make_descriptor_attr";
+  let mnemonic = "make_descriptor_attr";
+  let cppNamespace = "::mlir::triton";
+
+  let description = [{
+    Additional attributes for MakeTensorDescOp.
+  }];
+
+  // fp4Padded: Indicates that this encoding represents a mixed-precision fp4 operand in MMAv5 scaled dot, which needs
+  // to be in the special padded layout as described in https://docs.nvidia.com/cuda/parallel-thread-execution/#packing-format-used-for-matrix-a-and-b-by-kind-mxf8f6f4-in-shared-memory
+  let parameters = (
+    ins
+    "bool":$fp4Padded,
+    "unsigned":$swizzlingByteWidth
+  );
+
+  let assemblyFormat = "`<` struct(params) `>`";
+}
+
+def TT_DescriptorLoadAttr : AttrDef<Triton_Dialect, "DescriptorLoad"> {
+  let attrName = "triton.gpu.descriptor_load_attr";
+  let mnemonic = "descriptor_load_attr";
+  let cppNamespace = "::mlir::triton";
+
+  let description = [{
+    Additional attributes for ExperimentalDescriptorLoadOp.
+  }];
+
+  // fp4Padded: Indicates that this encoding represents a mixed-precision fp4 operand in MMAv5 scaled dot, which needs
+  // to be in the special padded layout as described in https://docs.nvidia.com/cuda/parallel-thread-execution/#packing-format-used-for-matrix-a-and-b-by-kind-mxf8f6f4-in-shared-memory
+  // twoCTAs: Indicates that the result of the load is consumed by an MMAv5 op with the 2CTA mode.
+  let parameters = (
+    ins
+    "bool":$fp4Padded,
+    "bool":$twoCTAs
```

Comment:
I don't think twoCTAs should be a special case, we should encode how the load is going to be distributed across CTAs